packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -214,7 +214,7 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
           ) {
             possibleFirstIndex -= 1;
           }
-          max = possibleFirstIndex * itemExtent;
+          max = (possibleFirstIndex + 1) * itemExtent;
```

Comment:
Good catch! Did you add a test that specifically covers this fix?

Reply:
A large number of the tests cover this case now that we're clamping when the content changes height.

Reply:
(to be 100% honest I'm not sure how to directly test this case...)